docs/user-guide/ldap-configuration.md

```diff
@@ -30,19 +29,22 @@ On this page, you'll find configuration options divided into three parts:
 
 The General Options include the following settings:
 
-- **Name:** Currently, only the name `ldap-iomete` is accepted.
-- **Enabled** If the provider is disabled, it won't be used for queries, and imported users will be read-only until the provider is re-enabled.
-- **Username LDAP attribute** Name of LDAP attribute, which is mapped as IOMETE username.
-- **RDN LDAP attribute** The LDAP attribute used as the top attribute in a typical user DN is called the RDN. It's usually the same as the Username LDAP attribute, but it doesn't have to be. For instance, in Active Directory, 'cn' is commonly used as the RDN attribute, even though the username attribute might be 'sAMAccountName'.
-- **UUID LDAP attribute** The LDAP attribute used as a unique identifier (UUID) for objects varies among LDAP server vendors. For most, it's 'entryUUID,' but for Active Directory, it's 'objectGUID.' If your LDAP server doesn't support UUID, you can use other unique attributes like 'uid' or 'entryDN' for LDAP users in the tree.
-- **User Object Classes** All user LDAP objectClass attributes are separated by commas, like 'inetOrgPerson, organizationalPerson'. IOMETE will write new users to LDAP with these object classes, and existing LDAP user records are only identified if they have all these object classes.
-- **Connection URL** Connection URL to your LDAP server
-- **Users DN** Here is the LDAP tree where your users are located: 'ou=users,dc=example,dc=com'. This is the parent DN of LDAP users. For instance, a typical user's DN might look like 'uid=john,ou=users,dc=example,dc=com'.
-- **Custom User LDAP Filter** Add an extra LDAP Filter to refine user search. Keep it blank if not needed. Start with '(' and end with ')' for correctness.
-- **Bind DN** LDAP admin DN, which IOMETE will use to access the LDAP server.
-- **Bind Credential** LDAP admin password. This field can get its value from the vault using the $\{vault.ID} format.
 
-<Img src="/img/user-guide/ldap/ldap-configuration-general-options.png" alt="LDAP configuration general inputs" maxWidth="500px"/>
+- **Connection URL**:
+  The URL to connect to your LDAP server, `for example, ldap://openldap.infra.svc:389` 
```

Comment:
"for example" and "uch as" should be outside the backticks:

![CleanShot 2024-10-08 at 20 55 06](https://github.com/user-attachments/assets/a70903ee-8cb4-400b-8218-8af30e65ec99)